CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,176 @@
+# W3C Code of Ethics and Professional Conduct
+
+(Originally [published at w3.org](https://www.w3.org/Consortium/cepc/))
+
```

Comment:
Should we duplicate the W3C code of conduct, or merely link to it? Duplicating means we might get out of sync if upstream is updated.

Reply:
That's a good question!

# Linking

## Pro

  * Less maintenance concern

## Con

  * Requires more clicks to get to content
  * W3C site experience different than GitHub/repo experience
  * Potential to have [slightly off-putting message](https://github.com/facebookresearch/TensorComprehensions/blob/master/CODE_OF_CONDUCT.md)

# Including

## Pro

  * Easy for new contributors to access the content
  * No need for context-switching

## Con

  * Need to periodically update the content as the CEPC changes

Given that the CEPC hasn't been updated in nearly 3 years, it's probably safe to include it as markdown and update it the next time there's an edit.

Reply:
I mean, we could easily just have a less off-putting message than that example does. :) I'm mostly in favor of linking, fwiw.